services/backend/src/services/analyticsService.js

```diff
@@ -1,11 +1,11 @@
 const moment = require('moment')
+const { facultyCodes } = require('../../config/organisationConstants')
 
 const { redisClient } = require('./redis')
 
 // Only new bachelor, masters and doctoral programmes get their data updated in redis every night, use redis for them
 const isUpdatedNewProgramme = code => code.includes('KH') || code.includes('MH') || /^(T)[0-9]{6}$/.test(code)
-const isFaculty = code =>
-  ['H10', 'H20', 'H30', 'H40', 'H50', 'H55', 'H57', 'H60', 'H70', 'H74', 'H80', 'H90'].includes(code)
+const isFaculty = code => facultyCodes.includes(code)
```

Comment:
The array `facultyCodes` also includes the codes in `ignoredFacultyCodes`. This should be taken into account.

Reply:
Edited to filter out `ignoredFacultyCodes`